docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,237 @@
+  Headless login only supports following `tsh` commands:
+  
+  - `tsh ls`
+  - `tsh ssh`
+  - `tsh scp`
+
+  In the future, headless login will be extended to other `tsh` commands.
+</Admonition>
+
+## Prerequisites
+
+- A v12.2+ Teleport cluster with WebAuthn configured.
+  See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
+- WebAuthn hardware device, such as YubiKey.
+- A Web browser with [WebAuthn support](
+  https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/).
+
+A v12.2+ Teleport cluster capable of WebAuthn is automatically capable of Headless login.
+
+## Step 1/2. Initiate headless login
+
+Run a headless tsh command with the `--headless` flag. This will initiate headless login, printing a URL and `tsh` command.
+
+```code
+$ tsh ls --headless --proxy=proxy.example.com --user=alice
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/86172f78-af7c-5935-a7c1-ed06b94f17dc
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=alice --proxy=proxy.example.com 86172f78-af7c-5935-a7c1-ed06b94f17dc
+```
+
+## Step 2/2. Approve headless Login
+
+To approve the headless login, click or copy+paste the URL printed by `tsh` in your local web browser. You will be prompted to approve the login with WebAuthn verification. Once approved, your initial `tsh --headless <command>` should continue as if you had logged in locally.
+
+Unlike a standard login session, headles login sessions are only available for the lifetime of a single `tsh` request. This means that for each `tsh --headless` command, you will need to go through the headless login flow:
+
+```code 
+$ tsh ls --headless --proxy=proxy.example.com --user=alice
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/86172f78-af7c-5935-a7c1-ed06b94f17dc
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=alice --proxy=proxy.example.com 86172f78-af7c-5935-a7c1-ed06b94f17dc
+# # User approves through link
+# Node Name Address        Labels                                                                             
+# --------- -------------- -----------
+# server01  127.0.0.1:3022 arch=x86_64
+#
+$ tsh ssh --headless --proxy=proxy.example.com --user=alice server01
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/864cccd9-2425-46d9-a9f2-636387e66ebf
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=alice --proxy=proxy.example.com 864cccd9-2425-46d9-a9f2-636387e66ebf
+# # User approves through link
+```
+
+## Additional Options
+
+### Set headless login as default
+
+To make headless login the default authentication method for your Teleport Cluster, add `connector_name: headless` to your cluster configuration:
+
+
+<ScopedBlock scope={["oss", "enterprise"]}>
```

Comment:
It's worth noting that users tend not to realize that you can switch scopes in the docs. We should reserve `ScopedBlock` components for situations where it's okay if a user reads the page with the wrong scope selected (i.e., minor cosmetic differences). 

What would you think of including only the dynamic configuration example in the how-to guide—since this works for all Teleport deployments—and making sure the configuration reference (`docs/pages/reference/config.mdx`) mentions this setting so users who need a static config know how to set it?